lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -585,7 +561,6 @@ def export_to_file(self, trans, id):
         """
         Get the latest Workflow for the StoredWorkflow identified by `id` and
         encode it as a json string that can be imported back into Galaxy
```

Comment:
I just noticed that this method has the same docstring of the one above.

```suggestion
        export it to a JSON file that can be imported back into Galaxy.

```